about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>MWC Barcelona About me</title>
+</head>
+
+<body>
+    <header class="firstPage">
+        <nav id="firstDesktopNav">
+            <a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            <a href="https://github.com/algerina">My Page</a>
+            <a href="#">Logout</a>
+        </nav>
+        <nav class="secondDesktopNav">
+            <a href="./index.html"><img src="./img/logo4.png" alt=""></a>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="">Join</a>
+            <a href="">Sponsor</a>
+        </nav>
+        <!--mobile navbar-->
+        <section class="mobile-nav">
+            <nav id="mobileSidebar" class="sidebar">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">X</a>
+                <a href="#">About</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+                <a href="./index.html">Home</a>
+            </nav>
+            <div id="main">
+                <button class="openbtn" onclick="openNav()">☰</button>
+            </div>
+        </section>
```

Comment:
- [ ] On mobile view, there should be a home button that will redirect to the index page.

Reply:
I already implemented this feature in the Hamburger side menu.

Reply:
- [ ] The links on your bottom navbar are too spaced out. Please try and reduce the spacing between them.